### PR TITLE
Don't show values in case calibration data is not reasonable.

### LINKED ISSUE #633: xdrip showing 212 and up while real bg was 54
This very scary thing was reported in a local whatsapp group a couple of days ago.

Long story short: 
Old sensor has stopped functioning. Tried calibration a few times. 
Sensor was replaced. Not stopped, not restarted, After two hours at 00:52 am, mother is awake, comming to do a big insulin doze. Very lucky, teenage daughter is awake and saying she is low. Measured by hand 54.

Calibration data:

time                                     BG              slope                                   intercept                       raw data
"2018-09-19 11:05:42"	"149.0"	"1.08440376574106"	"31.9426385098509"	"122.23528495"
"2018-09-20 16:29:20"	"258.0"	"1.16322807074784"	"28.553673821963"	"191.2941033"
"2018-09-22 14:07:38"	"304.0"	"1.07617139555535"	"35.5346328261269"	"255.9999808"
"2018-09-27 15:03:43"	"62.0"	"1.0"	"6.11765125"	"55.88234875"
"2018-10-16 14:57:32"	"199.0"	"1.0"	"138.7647104"	"60.2352896"
"2018-10-16 19:53:56"	"297.0"	"1.5"	"123.490643559024"	"121.05881445"
"2018-10-17 00:52:18"	"54.0"	"1.5"	"-34.5181463621557"	"59.41176025"
"2018-10-17 07:13:44"	"225.0"	"0.505575198398265"	"113.992377308007"	"259.9999805"
"2018-10-17 07:14:32"	"225.0"	"0.55"	"95.043928027153"	"259.9999805"
"2018-10-17 07:15:58"	"225.0"	"0.55"	"98.9977340642363"	"229.76468865"

BG data:
time                                       raw data              age adjusted raw data  calculated value
"2018-10-16 09:22:07"	"116.58822655"	"116.58822655"	"122.7058778"
"2018-10-16 09:27:08"	"113.6470503"	"113.6470503"	"119.76470155"
"2018-10-16 09:32:07"	"112.7058739"	"112.7058739"	"118.82352515"
"2018-10-16 09:37:07"	"114.1176385"	"114.1176385"	"120.23528975"
"2018-10-16 09:42:05"	"119.2941087"	"119.2941087"	"125.41175995"
"2018-10-16 09:51:30"	"119.17646165"	"119.17646165"	"125.2941129"
"2018-10-16 09:56:32"	"114.3529326"	"114.3529326"	"120.47058385"
"2018-10-16 10:01:29"	"114.8235208"	"114.8235208"	"120.94117205"
"2018-10-16 10:06:29"	"112.58822685"	"112.58822685"	"118.7058781"
"2018-10-16 10:11:30"	"110.1176388"	"110.1176388"	"116.23529005"
"2018-10-16 10:16:10"	"107.88234485"	"107.88234485"	"113.9999961"
"2018-10-16 10:22:10"	"106.8235214"	"106.8235214"	"112.94117265"
"2018-10-16 10:27:09"	"104.4705804"	"104.4705804"	"110.58823165"
"2018-10-16 10:32:09"	"102.94116875"	"102.94116875"	"109.05882"
"2018-10-16 10:37:09"	"99.2941102"	"99.2941102"	"105.41176145"
"2018-10-16 10:42:09"	"97.52940445"	"97.52940445"	"103.6470557"
"2018-10-16 10:47:09"	"97.99999265"	"97.99999265"	"104.1176439"
"2018-10-16 10:55:50"	"97.29411035"	"97.29411035"	"103.4117616"
"2018-10-16 11:00:50"	"93.6470518"	"93.6470518"	"99.76470305"
"2018-10-16 11:05:50"	"90.8235226"	"90.8235226"	"96.94117385"
"2018-10-16 11:10:50"	"85.76469945"	"85.76469945"	"91.8823507"
"2018-10-16 11:15:50"	"81.05881745"	"81.05881745"	"87.1764687"
"2018-10-16 11:20:50"	"77.4117589"	"77.4117589"	"83.52941015"
"2018-10-16 11:25:50"	"71.7647005"	"71.7647005"	"77.88235175"
"2018-10-16 11:30:50"	"65.52940685"	"65.52940685"	"71.6470581"
"2018-10-16 11:35:50"	"56.7058781"	"56.7058781"	"62.82352935"
"2018-10-16 11:40:50"	"46.70587885"	"46.70587885"	"52.8235301"
"2018-10-16 11:45:50"	"38.70587945"	"38.70587945"	"44.8235307"
"2018-10-16 11:50:50"	"32.82352695"	"32.82352695"	"39.0"
"2018-10-16 11:55:50"	"28.82352725"	"28.82352725"	"39.0"
"2018-10-16 12:00:51"	"25.76470395"	"25.76470395"	"39.0"
"2018-10-16 12:05:50"	"25.6470569"	"25.6470569"	"39.0"
"2018-10-16 12:10:50"	"30.47058595"	"30.47058595"	"41.6485866272982"
"2018-10-16 12:15:50"	"32.4705858"	"32.4705858"	"44.4060049158684"
"2018-10-16 12:20:50"	"38.70587945"	"38.70587945"	"52.8591330668099"
"2018-10-16 12:25:52"	"45.8823495"	"45.8823495"	"62.5383017819337"
"2018-10-16 12:30:52"	"53.29411365"	"53.29411365"	"72.3576404972899"
"2018-10-16 12:35:53"	"57.8823486"	"57.8823486"	"79.3664214807962"
"2018-10-16 12:40:50"	"62.1176424"	"62.1176424"	"85.9811849984938"
"2018-10-16 12:45:49"	"65.76470095"	"65.76470095"	"91.984652878369"
"2018-10-16 12:54:29"	"65.76470095"	"65.76470095"	"94.4974404631651"
"2018-10-16 13:03:05"	"63.64705405"	"63.64705405"	"95.0222958294625"
"2018-10-16 13:08:06"	"62.70587765"	"62.70587765"	"96.6702945489205"
"2018-10-16 13:13:05"	"62.8235247"	"62.8235247"	"99.310818103389"
"2018-10-16 13:18:05"	"61.52940715"	"61.52940715"	"100.650555174253"
"2018-10-16 13:26:43"	"56.9411722"	"56.9411722"	"98.9945186125698"
"2018-10-16 13:31:42"	"55.5294076"	"55.5294076"	"100.279304062826"
"2018-10-16 13:36:43"	"54.70587825"	"54.70587825"	"102.111969084393"
"2018-10-16 13:41:43"	"53.99999595"	"53.99999595"	"104.060119244602"
"2018-10-16 13:46:42"	"52.35293725"	"52.35293725"	"105.179512395445"
"2018-10-16 13:51:44"	"49.29411395"	"49.29411395"	"105.071624048867"
"2018-10-16 14:01:00"	"46.70587885"	"46.70587885"	"105.414146062945"
"2018-10-16 14:05:57"	"45.99999655"	"45.99999655"	"107.428594838143"
"2018-10-16 14:10:58"	"44.82352605"	"44.82352605"	"109.044519125287"
"2018-10-16 14:19:36"	"43.2941144"	"43.2941144"	"110.371638966061"
"2018-10-16 14:24:37"	"43.9999967"	"43.9999967"	"113.624178795471"
"2018-10-16 14:29:37"	"44.11764375"	"44.11764375"	"116.36991536938"
"2018-10-16 14:34:49"	"44.4705849"	"44.4705849"	"119.312031638838"
"2018-10-16 14:39:49"	"47.05882"	"47.05882"	"124.1154311933"
"2018-10-16 14:44:50"	"48.9411728"	"48.9411728"	"128.302307450575"
"2018-10-16 14:49:34"	"55.2941135"	"55.2941135"	"136.142569114392"
"2018-10-16 14:54:36"	"60.2352896"	"60.2352896"	"142.75266673231"
"2018-10-16 14:59:35"	"69.29411245"	"69.29411245"	"150.146817836377"
"2018-10-16 15:04:35"	"76.4705825"	"76.4705825"	"215.2352929"
"2018-10-16 15:09:34"	"85.52940535"	"85.52940535"	"224.29411575"
"2018-10-16 15:14:34"	"89.411758"	"89.411758"	"228.1764684"
"2018-10-16 15:19:35"	"95.88234575"	"95.88234575"	"234.64705615"
"2018-10-16 15:24:35"	"100.4705807"	"100.4705807"	"239.2352911"
"2018-10-16 15:29:34"	"102.3529335"	"102.3529335"	"241.1176439"
"2018-10-16 15:34:34"	"105.882345"	"105.882345"	"244.6470554"
"2018-10-16 15:39:34"	"108.82352125"	"108.82352125"	"247.58823165"
"2018-10-16 15:44:33"	"110.1176388"	"110.1176388"	"248.8823492"
"2018-10-16 15:49:33"	"112.35293275"	"112.35293275"	"251.11764315"
"2018-10-16 15:54:34"	"114.94116785"	"114.94116785"	"253.70587825"
"2018-10-16 15:59:34"	"116.2352854"	"116.2352854"	"254.9999958"
"2018-10-16 16:04:33"	"114.3529326"	"114.3529326"	"253.117643"
"2018-10-16 16:09:33"	"111.0588152"	"111.0588152"	"249.8235256"
"2018-10-16 16:14:33"	"108.58822715"	"108.58822715"	"247.35293755"
"2018-10-16 16:19:34"	"107.88234485"	"107.88234485"	"246.64705525"
"2018-10-16 16:24:34"	"108.11763895"	"108.11763895"	"246.88234935"
"2018-10-16 16:29:33"	"109.76469765"	"109.76469765"	"248.52940805"
"2018-10-16 16:34:33"	"109.76469765"	"109.76469765"	"248.52940805"
"2018-10-16 16:39:31"	"107.5294037"	"107.5294037"	"246.2941141"
"2018-10-16 16:44:30"	"103.529404"	"103.529404"	"242.2941144"
"2018-10-16 16:49:35"	"100.82352185"	"100.82352185"	"239.58823225"
"2018-10-16 16:54:30"	"98.70587495"	"98.70587495"	"237.47058535"
"2018-10-16 16:59:31"	"97.76469855"	"97.76469855"	"236.52940895"
"2018-10-16 17:04:30"	"99.88234545"	"99.88234545"	"238.64705585"
"2018-10-16 17:13:07"	"101.29411005"	"101.29411005"	"240.05882045"
"2018-10-16 17:18:06"	"102.23528645"	"102.23528645"	"239.944324283554"
"2018-10-16 17:23:07"	"104.7058745"	"104.7058745"	"241.33340136977"
"2018-10-16 17:28:01"	"104.2352863"	"104.2352863"	"239.801603803894"
"2018-10-16 17:33:03"	"105.1764627"	"105.1764627"	"239.656962840554"
"2018-10-16 17:38:02"	"105.99999205"	"105.99999205"	"239.387907627911"
"2018-10-16 17:43:00"	"106.94116845"	"106.94116845"	"239.224195466536"
"2018-10-16 17:48:01"	"110.94116815"	"110.94116815"	"241.997497017714"
"2018-10-16 17:53:00"	"111.9999916"	"111.9999916"	"241.910828710524"
"2018-10-16 17:58:00"	"113.76469735"	"113.76469735"	"242.485748307898"
"2018-10-16 18:03:01"	"117.64705"	"117.64705"	"245.049122818585"
"2018-10-16 18:10:33"	"118.23528525"	"118.23528525"	"244.46725831393"
"2018-10-16 18:15:34"	"117.05881475"	"117.05881475"	"242.225272048498"
"2018-10-16 18:20:37"	"115.0588149"	"115.0588149"	"239.228043626029"
"2018-10-16 18:25:32"	"113.29410915"	"113.29410915"	"236.46980050318"
"2018-10-16 18:30:33"	"113.8823444"	"113.8823444"	"235.898394397447"
"2018-10-16 18:35:32"	"116.11763835"	"116.11763835"	"236.830090206549"
"2018-10-16 18:40:34"	"121.29410855"	"121.29410855"	"240.418124695004"
"2018-10-16 18:45:33"	"122.23528495"	"122.23528495"	"240.115377347957"
"2018-10-16 18:50:33"	"123.17646135"	"123.17646135"	"239.802786801925"
"2018-10-16 18:55:33"	"125.52940235"	"125.52940235"	"240.744469672121"
"2018-10-16 19:00:34"	"127.41175515"	"127.41175515"	"241.242633139529"
"2018-10-16 19:05:32"	"127.2941081"	"127.2941081"	"239.951195135255"
"2018-10-16 19:10:33"	"130.47057845"	"130.47057845"	"241.558916171681"
"2018-10-16 19:15:33"	"126.1176376"	"126.1176376"	"236.548952314153"
"2018-10-16 19:20:33"	"128.2352845"	"128.2352845"	"237.209201089105"
"2018-10-16 19:25:33"	"128.4705786"	"128.4705786"	"236.220873039966"
"2018-10-16 19:30:33"	"125.29410825"	"125.29410825"	"232.30002129144"
"2018-10-16 19:35:33"	"122.94116725"	"122.94116725"	"229.115340501918"
"2018-10-16 19:40:33"	"121.8823438"	"121.8823438"	"227.053136480082"
"2018-10-16 19:46:00"	"121.05881445"	"121.05881445"	"225.20038816091"
"2018-10-16 19:54:40"	"115.17646195"	"115.17646195"	"220.240835565815"
"2018-10-16 19:59:40"	"116.7058736"	"116.7058736"	"298.549453959024"
"2018-10-16 20:04:40"	"116.11763835"	"116.11763835"	"297.667101084024"
"2018-10-16 20:09:40"	"118.5882264"	"118.5882264"	"301.372983159024"
"2018-10-16 20:14:41"	"114.3529326"	"114.3529326"	"295.020042459024"
"2018-10-16 20:19:42"	"110.8235211"	"110.8235211"	"289.725925209024"
"2018-10-16 20:24:41"	"111.17646225"	"111.17646225"	"290.255336934024"
"2018-10-16 20:33:18"	"106.5882273"	"106.5882273"	"283.372984509024"
"2018-10-16 20:38:38"	"104.4705804"	"104.4705804"	"280.196514159024"
"2018-10-16 20:43:38"	"101.8823453"	"101.8823453"	"276.314161509024"
"2018-10-16 20:51:08"	"96.35293395"	"96.35293395"	"268.020044484024"
"2018-10-16 20:56:34"	"93.29411065"	"93.29411065"	"263.431809534024"
"2018-10-16 21:03:34"	"88.35293455"	"88.35293455"	"256.020045384024"
"2018-10-16 21:08:35"	"84.2352878"	"84.2352878"	"249.843575259024"
"2018-10-16 21:13:35"	"79.41175875"	"79.41175875"	"242.608281684024"
"2018-10-16 21:28:02"	"0.012"	"0.012"	"12.0"
"2018-10-16 21:33:38"	"77.76470005"	"77.76470005"	"82.1289037128443"
"2018-10-16 21:43:02"	"186.70586835"	"186.70586835"	"400.0"
"2018-10-16 21:48:38"	"198.5882204"	"198.5882204"	"263.364184237844"
"2018-10-16 21:58:02"	"175.0588104"	"175.0588104"	"386.078859159024"
"2018-10-16 22:03:38"	"165.1764582"	"165.1764582"	"213.246540937844"
"2018-10-16 22:13:02"	"151.88234155"	"151.88234155"	"351.314155884024"
"2018-10-16 22:22:02"	"119.7646969"	"119.7646969"	"303.137688909024"
"2018-10-16 22:27:02"	"116.82352065"	"116.82352065"	"295.115574186362"
"2018-10-16 22:32:02"	"111.5294034"	"111.5294034"	"283.688377616284"
"2018-10-16 22:37:02"	"110.1176388"	"110.1176388"	"278.072277055948"
"2018-10-16 22:42:01"	"101.176463"	"101.176463"	"261.548863059205"
"2018-10-16 22:47:01"	"89.411758"	"89.411758"	"241.16590710138"
"2018-10-16 22:52:01"	"77.8823471"	"77.8823471"	"221.395603123396"
"2018-10-16 22:57:02"	"71.17646525"	"71.17646525"	"208.734881627244"
"2018-10-16 23:02:01"	"65.99999505"	"65.99999505"	"198.382092011935"
"2018-10-16 23:07:02"	"61.76470125"	"61.76470125"	"189.463170045974"
"2018-10-16 23:12:02"	"57.76470155"	"57.76470155"	"180.969024140394"
"2018-10-16 23:17:02"	"53.76470185"	"53.76470185"	"172.568816194545"
"2018-10-16 23:22:02"	"49.76470215"	"49.76470215"	"164.262546208425"
"2018-10-16 23:27:02"	"44.58823195"	"44.58823195"	"154.465095707988"
"2018-10-16 23:32:01"	"39.999997"	"39.999997"	"145.574864013068"
"2018-10-16 23:37:02"	"36.4705855"	"36.4705855"	"138.194125379845"
"2018-10-16 23:42:02"	"38.23529125"	"38.23529125"	"137.842811836011"
"2018-10-16 23:47:02"	"41.76470275"	"41.76470275"	"139.744846350663"
"2018-10-16 23:52:01"	"44.35293785"	"44.35293785"	"140.351157274392"
"2018-10-16 23:57:01"	"49.29411395"	"49.29411395"	"143.901148890395"
"2018-10-17 00:02:02"	"51.2941138"	"51.2941138"	"143.614055414451"
"2018-10-17 00:07:03"	"50.3529374"	"50.3529374"	"139.593484890378"
"2018-10-17 00:12:03"	"51.41176085"	"51.41176085"	"138.07835841214"
"2018-10-17 00:17:02"	"56.2352899"	"56.2352899"	"141.16868413337"
"2018-10-17 00:22:01"	"48.11764345"	"48.11764345"	"128.380971619137"
"2018-10-17 00:27:01"	"43.17646735"	"43.17646735"	"119.616131036156"
"2018-10-17 00:32:01"	"48.4705846"	"48.4705846"	"123.195452885657"
"2018-10-17 00:37:01"	"54.70587825"	"54.70587825"	"127.763818516375"
"2018-10-17 00:42:01"	"60.58823075"	"60.58823075"	"131.772380729239"
"2018-10-17 00:47:01"	"64.58823045"	"64.58823045"	"133.460258114774"
"2018-10-17 00:52:02"	"59.41176025"	"59.41176025"	"124.522063657747"
"2018-10-17 00:57:02"	"51.9999961"	"51.9999961"	"116.015340136998"
"2018-10-17 01:02:02"	"59.64705435"	"59.64705435"	"54.9524351628443"
"2018-10-17 01:10:38"	"51.764702"	"51.764702"	"43.1289066378443"
"2018-10-17 01:15:38"	"51.88234905"	"51.88234905"	"43.3053772128443"
"2018-10-17 01:20:38"	"57.05881925"	"57.05881925"	"51.0700825128443"
"2018-10-17 01:25:38"	"66.1176421"	"66.1176421"	"64.6583167878443"
"2018-10-17 01:30:39"	"90.70587555"	"90.70587555"	"101.540666962844"
"2018-10-17 01:35:39"	"107.7646978"	"107.7646978"	"127.128900337844"
"2018-10-17 01:40:39"	"121.1764615"	"121.1764615"	"147.246545887844"
"2018-10-17 01:45:39"	"130.94116665"	"130.94116665"	"161.893603612844"
"2018-10-17 01:50:39"	"131.5294019"	"131.5294019"	"162.775956487844"
"2018-10-17 01:55:38"	"131.64704895"	"131.64704895"	"162.952427062844"
"2018-10-17 02:00:39"	"134.117637"	"134.117637"	"166.658309137844"
"2018-10-17 02:05:39"	"137.8823426"	"137.8823426"	"172.305367537844"
"2018-10-17 02:10:38"	"142.94116575"	"142.94116575"	"179.893602262844"
"2018-10-17 02:15:39"	"142.94116575"	"142.94116575"	"179.893602262844"
"2018-10-17 02:20:38"	"159.2941057"	"159.2941057"	"204.423012187844"
"2018-10-17 02:25:39"	"171.88234005"	"171.88234005"	"223.305363712844"
"2018-10-17 02:30:38"	"178.5882219"	"178.5882219"	"233.364186487844"
"2018-10-17 02:35:38"	"173.4117517"	"173.4117517"	"225.599481187844"
"2018-10-17 02:40:37"	"173.52939875"	"173.52939875"	"225.775951762844"
"2018-10-17 02:45:37"	"166.1176346"	"166.1176346"	"214.658305537844"
"2018-10-17 02:50:37"	"155.17645895"	"155.17645895"	"198.246542062844"
"2018-10-17 02:55:38"	"146.94116545"	"146.94116545"	"185.893601812844"
"2018-10-17 03:00:38"	"154.70587075"	"154.70587075"	"197.540659762844"
"2018-10-17 03:05:38"	"161.6470467"	"161.6470467"	"207.952423687844"
"2018-10-17 03:10:38"	"162.47057605"	"162.47057605"	"209.187717712844"
"2018-10-17 03:15:38"	"167.7646933"	"167.7646933"	"217.128893587844"
"2018-10-17 03:20:38"	"175.17645745"	"175.17645745"	"228.246539812844"
"2018-10-17 03:25:38"	"185.99998605"	"185.99998605"	"244.481832712844"
"2018-10-17 03:30:38"	"195.0588089"	"195.0588089"	"258.070066987844"
"2018-10-17 03:35:38"	"209.411749"	"209.411749"	"279.599477137844"
"2018-10-17 03:40:38"	"228.35292405"	"228.35292405"	"308.011239712844"
"2018-10-17 03:45:37"	"213.76468985"	"213.76468985"	"286.128888412844"
"2018-10-17 03:50:37"	"203.9999847"	"203.9999847"	"271.481830687844"
"2018-10-17 03:55:38"	"210.23527835"	"210.23527835"	"280.834771162844"
"2018-10-17 04:00:39"	"252.7058634"	"252.7058634"	"344.540648737844"
"2018-10-17 04:05:39"	"241.76468775"	"241.76468775"	"328.128885262844"
"2018-10-17 04:10:39"	"237.647041"	"237.647041"	"321.952415137844"
"2018-10-17 04:15:38"	"237.52939395"	"237.52939395"	"321.775944562844"
"2018-10-17 04:20:37"	"277.8823321"	"277.8823321"	"382.305351787844"
"2018-10-17 04:25:37"	"290.3529194"	"290.3529194"	"400.0"
"2018-10-17 04:30:37"	"300.58821275"	"300.58821275"	"400.0"
"2018-10-17 04:35:38"	"289.8823312"	"289.8823312"	"400.0"
"2018-10-17 04:40:38"	"271.2940973"	"271.2940973"	"372.422999587844"
"2018-10-17 04:45:37"	"272.4705678"	"272.4705678"	"366.161018757818"
"2018-10-17 04:50:37"	"265.76468595"	"265.76468595"	"347.223834939727"
"2018-10-17 04:55:37"	"271.9999796"	"271.9999796"	"346.570608826066"
"2018-10-17 05:00:37"	"274.1176265"	"274.1176265"	"340.779598611903"
"2018-10-17 05:05:38"	"269.1764504"	"269.1764504"	"327.416980221329"
"2018-10-17 05:10:37"	"266.70586235"	"266.70586235"	"318.004090175922"
"2018-10-17 05:15:37"	"264.82350955"	"264.82350955"	"310.107568699"
"2018-10-17 05:20:38"	"285.1764492"	"285.1764492"	"324.527942176908"
"2018-10-17 05:25:37"	"278.5882144"	"278.5882144"	"312.818113820574"
"2018-10-17 05:30:37"	"271.41174435"	"271.41174435"	"301.762864663374"
"2018-10-17 05:35:37"	"267.64703875"	"267.64703875"	"294.69222388477"
"2018-10-17 05:40:37"	"264.7058625"	"264.7058625"	"289.052684089674"
"2018-10-17 05:45:37"	"262.23527445"	"262.23527445"	"284.425438050483"
"2018-10-17 05:50:36"	"277.8823321"	"277.8823321"	"294.184583397376"
"2018-10-17 05:55:36"	"265.6470389"	"265.6470389"	"283.003329170493"
"2018-10-17 06:00:38"	"273.76468535"	"273.76468535"	"287.448237258606"
"2018-10-17 06:05:37"	"257.6470395"	"257.6470395"	"274.77272009297"
"2018-10-17 06:10:36"	"244.11762875"	"244.11762875"	"264.625359721639"
"2018-10-17 06:15:36"	"235.7646882"	"235.7646882"	"258.580563989858"
"2018-10-17 06:20:39"	"230.1176298"	"230.1176298"	"254.727138201471"
"2018-10-17 06:25:37"	"225.6470419"	"225.6470419"	"251.913160145807"
"2018-10-17 06:30:38"	"221.76468925"	"221.76468925"	"249.678251917379"
"2018-10-17 06:35:38"	"218.1176307"	"218.1176307"	"247.739462002744"
"2018-10-17 06:40:36"	"247.7646873"	"247.7646873"	"268.774562728149"
"2018-10-17 06:45:36"	"254.3529221"	"254.3529221"	"274.199289506164"
"2018-10-17 06:50:36"	"232.35292375"	"232.35292375"	"259.917290413744"
"2018-10-17 06:55:36"	"216.58821905"	"216.58821905"	"249.851950696051"
"2018-10-17 07:00:37"	"205.6470434"	"205.6470434"	"243.055238062766"
"2018-10-17 07:05:36"	"258.94115705"	"258.94115705"	"282.080584048354"
"2018-10-17 07:10:37"	"259.9999805"	"259.9999805"	"284.165016959108"
"2018-10-17 07:15:37"	"229.76468865"	"229.76468865"	"263.424653263492"
"2018-10-17 07:20:36"	"219.88233645"	"219.88233645"	"256.209267803264"
"2018-10-17 07:25:36"	"209.1764549"	"209.1764549"	"214.044784259236"
"2018-10-17 07:30:37"	"197.99998515"	"197.99998515"	"207.897725896736"
"2018-10-17 07:39:13"	"237.647041"	"237.647041"	"229.703606614236"
"2018-10-17 07:44:14"	"253.05880455"	"253.05880455"	"238.180076566736"
"2018-10-17 07:49:14"	"247.5293932"	"247.5293932"	"235.138900324236"
"2018-10-17 07:54:13"	"224.9411596"	"224.9411596"	"222.715371844236"
"2018-10-17 07:59:14"	"213.99998395"	"213.99998395"	"216.697725236736"
"2018-10-17 08:04:14"	"219.9999835"	"219.9999835"	"219.997724989236"
"2018-10-17 08:09:14"	"197.8823381"	"197.8823381"	"207.833020019236"
"2018-10-17 08:14:14"	"169.6470461"	"169.6470461"	"192.303609419236"
"2018-10-17 08:19:13"	"151.5294004"	"151.5294004"	"182.338904284236"
"2018-10-17 08:24:14"	"141.76469525"	"141.76469525"	"176.968316451736"
"2018-10-17 08:29:14"	"168.2352815"	"168.2352815"	"191.527138889236"
"2018-10-17 08:34:14"	"191.88233855"	"191.88233855"	"204.533020266736"
"2018-10-17 08:39:14"	"192.58822085"	"192.58822085"	"204.921255531736"
"2018-10-17 08:44:14"	"164.4705759"	"164.4705759"	"189.456550809236"
"2018-10-17 08:49:14"	"143.0588128"	"143.0588128"	"177.680081104236"
"2018-10-17 08:54:14"	"129.411755"	"129.411755"	"170.174199314236"
"2018-10-17 08:59:13"	"121.52940265"	"121.52940265"	"165.838905521736"
"2018-10-17 09:04:14"	"116.11763835"	"116.11763835"	"162.862435156736"
"2018-10-17 09:11:54"	"118.70587345"	"118.70587345"	"164.285964461736"
"2018-10-17 09:16:54"	"138.94116605"	"138.94116605"	"175.415375391736"
"2018-10-17 09:21:54"	"140.7058718"	"140.7058718"	"176.385963554236"

### PR DESCRIPTION
**Pull request by tzachi-dar**
This is a proof of concept value which fixes #633. 

Very likely will not work, but to give a clue of what I think that we should do.

### CODE REVIEW

**Comment by jamorham** [comment]
I don't think this condition will ever be met as slope isn't likely to exceed ~1.5 due to `SlopeParameters`

**Comment by tzachi-dar** [comment]
My mistake. What I meant to say is that we should check the intercept.
This is the most problematic one.

File: app/src/main/java/com/eveningoutpost/dexdrip/Home.java [hunk 1]
```diff
@@ -2853,8 +2853,13 @@ private void displayCurrentInfo() {
                     toaststaticnext(msg);
                 }
             }
-
-            displayCurrentInfoFromReading(lastBgReading, predictive);
+            if(Calibration.IsReasnoble()) {
```
Comment (asienicki) [inline app/src/main/java/com/eveningoutpost/dexdrip/Home.java]: #typo IsReasnoble->IsReasonable

File: app/src/main/java/com/eveningoutpost/dexdrip/Home.java [hunk 2]
```diff
@@ -2853,8 +2853,13 @@ private void displayCurrentInfo() {
...
+                displayCurrentInfoFromReading(lastBgReading, predictive);
+            } else {
+                currentBgValueText.setText("Bad calibration value. Pleaes recalibrate again in an hour." +
```
Comment (asienicki) [inline app/src/main/java/com/eveningoutpost/dexdrip/Home.java]: text should be in resource

File: app/src/main/java/com/eveningoutpost/dexdrip/Models/Calibration.java [hunk 3]
```diff
@@ -1088,6 +1088,11 @@ public static Calibration lastValid() {
                 .orderBy("timestamp desc")
                 .executeSingle();
     }
+    
+    public static boolean IsReasnoble() {
+        Calibration calibration = lastValid();
+        return calibration.intercept < 35;
```
Comment (asienicki) [inline app/src/main/java/com/eveningoutpost/dexdrip/Models/Calibration.java]: what does it mean - magic int 35?

**Comment by Navid200** [comment]
@tzachi-dar Your first post in this pull request says "Very likely will not work, but to give a clue of what I think that we should do".
Do you think that can be accomplished if the pull request is closed?  Or, do you think we should keep it open?

**Comment by Navid200** [comment]
This is the last statement from 3 years ago:
>My mistake. What I meant to say is that we should check the intercept.
>This is the most problematic one.

Is this a valid pull request?
If not, would you please close it?

**Comment by tzachi-dar** [comment]
This task was achieved with another pr, so closing it.

**Pull request closed without merging**